lib/taskana-core/src/main/java/pro/taskana/configuration/DbSchemaCreator.java

```diff
@@ -20,6 +20,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DbSchemaCreator.class);
     private static final String SQL = "/sql";
     private static final String DB_SCHEMA = SQL + "/taskana-schema.sql";
+    private static final String DB_SCHEMA_POSTGRES = SQL + "/taskana-schema-postgres.sql";
```

Comment:
why is a different schema file needed? is this really necessary?

Reply:
Since postgres does not support the clob datatype i am forced to create a different schema file.
Additionally I used the datatype boolean instad of smallint for the postgres db. Explanation follows below